package.json

```diff
@@ -6,8 +6,12 @@
   "types": "build/src/index.d.ts",
   "files": [
     "build/src",
+    "build/tools/compileProtos.js",
     "protos"
   ],
+  "bin": {
+    "compileProtos": "./build/tools/compileProtos.js"
```

Comment:
totally a nit, but I usually use kebab case for bins.

Reply:
I do too but all the files in gax repo is camelCase, so I thought I'll go with it, otherwise it will look like an outlier :)